backend/sirius-components-formdescriptioneditors/src/main/java/org/eclipse/sirius/components/formdescriptioneditors/configuration/FormDescriptionEditorDescriptionRegistryConfigurer.java

```diff
@@ -66,6 +71,41 @@ public void addRepresentationDescriptions(IRepresentationDescriptionRegistry reg
                 .build();
         widgetsDescriptions.add(textfieldDescription);
 
+        UUID textareaId = UUID.nameUUIDFromBytes("FormDescriptionEditorTextArea".getBytes()); //$NON-NLS-1$
```

Comment:
What is the point of any of this? Unless I missed something, you don't use any of these widget descriptions so why create new ones? You don't use their instances, you don't use their classes anywhere.

In the form DSL, they make sense because they are used to parameterize the behavior of the edition and rendering of the widgets. Here, they have no other properties than `id` and `label` and neither `getId()` nor `getLabel()` is used (and I don't see why they should be).

Nobody is even calling `FormDescriptionEditorDescription.getWidgetDescriptions()` so I really think that you don't need any of these abstraction.

Reply:
Done